Riot/Modules/Room/RoomViewController.swift

```diff
@@ -154,6 +154,53 @@ extension RoomViewController {
         RiotSettings.shared.enableWysiwygTextFormatting.toggle()
         wysiwygInputToolbar?.textFormattingEnabled.toggle()
     }
+    
+    @objc func didChangeMaximisedState(_ state: Bool) {
+        if state {
+            var view: UIView!
+            // iPhone
+            if let navView = self.navigationController?.navigationController?.view {
+                view = navView
+            // iPad
+            } else if let navView = self.navigationController?.view {
+                view = navView
+            } else {
+                return
+            }
+            let originalRect = roomInputToolbarContainer.convert(roomInputToolbarContainer.frame, to: view)
+            self.roomInputToolbarContainer.removeFromSuperview()
+            let dimmingView = UIView(frame: view.bounds)
+            dimmingView.backgroundColor = UIColor.black.withAlphaComponent(0.5)
```

Comment:
Do we want something different depending on theme ? (probably more a question for design, I haven't checked how other screen behave in the application)

Reply:
Checked with Callum, he is fine by having the dimming view just be the same kind of the default iOS system dimming view, will check online if it is theme based and what are the expected opacity.

Reply:
done, is 0.12 for light mode and 0.29 for dark mode.